.github/workflows/release.yml

```
@@ -0,0 +1,54 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  goreleaser:
```

Thought that this PR was about releasing binaries as GitHub release assets _and_ pushing the Docker images.
Do we still want to do that or only do the latter?

For the latter, we're not using `goreleaser` anymore:
```suggestion
  release-docker:
```

---

I interpreted the original intention to be addressing #20, which is specifically a request for building an arm64 image, not binary.

We can also publish the binaries, but do we currently have an application for them?

---

Not really, so I'd say let's leave that out for now.